client/src/main/java/io/split/client/SplitClient.java

```diff
@@ -97,4 +97,29 @@
      * the any instances of SplitClient or SplitManager generated by the client's parent SplitFactory
      */
     void destroy();
+
+    /**
+     * Enqueue a new event to be sent to split data collection services
+     *
+     * Example:
+     *      client.track(“account”, “Split Software”, “checkout”)
+     *
+     * @param trafficType the type of the event
+     * @param key the identifier of the entity
+     * @param eventType the type of the event
+     */
+    void track(String trafficType, String key, String eventType);
+
+    /**
+     * Enqueue a new event to be sent to split data collection services
+     *
+     * Example:
+     *      client.track(“account”, “Split Software”, “checkout”, 200.00)
+     *
+     * @param trafficType the type of the event
+     * @param key the identifier of the entity
+     * @param eventType the type of the event
+     * @param value the value of the event
+     */
+    void track(String trafficType, String key, String eventType, double value);
```

Comment:
@sande this needs to return a boolean too

Reply:
done.